Sources/Runner/Commands/Runner.swift

```diff
@@ -121,25 +121,37 @@ func runDanger(logger: Logger) throws {
     // swift build; cat Fixtures/eidolon_609.json  | ./.build/debug/danger-swift
     // swiftlint:enable line_length
 
+    // This ain't optimal, but SwiftPM have _so much code_ around this.
+    // So maybe there's a better way
+    #if os(Linux)
+    let command = "which swiftc"
+    #else
+    let command = "xcrun --find swiftc"
+    #endif
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-default;
  File: Sources/Runner/Commands/Runner.swift;
  Line: 131;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)